pkgs/applications/audio/easyabc/default.nix

```diff
@@ -0,0 +1,56 @@
+{ lib, fetchFromGitHub, python39, makeWrapper, fluidsynth, soundfont-fluid, wrapGAppsHook, abcmidi, abcm2ps, ghostscript }:
+
+# requires python39 due to https://stackoverflow.com/a/71902541
+python39.pkgs.buildPythonApplication rec {
+  pname = "easyabc";
+  version = "1.3.8.6";
+
+  src = fetchFromGitHub {
+    owner = "jwdj";
+    repo = pname;
```

Comment:
```suggestion
    repo = "easyabc";
```

Reply:
There's lots of packages in nixpkgs having `repo = pname`, what's the criteria for choosing one style or the other?

Reply:
I don't understand why tbh? Being old is not a reason. Is there some standard guide for reviewers you're using that explains this?

Reply:
1. there is no real advantage on parameterization of a thing that will never change
2. being equal is not a good reason to parameterize something
3. sometimes `repo` is not perfectly equal to `pname`. 
     Examples: 
     1. it is insane to generate `openMSX` from `openmsx`.
     2. how precisely `pil` can generate `Imaging`?
4. it creates a useless dependency link between two otherwise unrelated statements, generating a more complicated data graph for no benefit.

Reply:
Applied in https://github.com/NixOS/nixpkgs/pull/237197/commits/5fa3e97bef6ce3b29331a1d0cfc006132d3c79ed